warnings.md

```diff
@@ -166,3 +166,9 @@ See jQuery-ui [commit](https://github.com/jquery/jquery-ui/commit/c0093b599fcd58
 **Cause:** The standard way to add new custom selectors through jQuery is `jQuery.expr.pseudos`. These two other aliases are deprecated, although they still work as of jQuery 3.0.
 
 **Solution:** Rename any of the older usage to `jQuery.expr.pseudos`. The functionality is identical.
+
+### JQMIGRATE: jQuery.fn.toggleClass( boolean ) is deprecated
```

Comment:
How about `[ boolean ]` to keep the error message short? The longer description already covers both cases.